pkg/cmd/container/create.go

```diff
@@ -27,6 +27,7 @@ import (
 	"path"
 	"path/filepath"
 	"runtime"
+	"slices"
```

Comment:
This doesn't work with Go 1.20.
I guess we can drop the support for Go 1.20 though.

Reply:
Switch to using or statement to maintain compatibility with 1.20